### ISSUE TITLE
Conflict typings @types/mocha and @types/jest

### ISSUE THREAD

**Issue by PiQx** [webdriverio/webdriverio#6966]
[//]: # NOTE: This repository only maintains packages that are listed in the [Readme](https://github.com/webdriverio/webdriverio/blob/main/README.md#packages). Please make sure that your issue is directly caused by one of these packages and if not file an issue in the correct 3rd party package repository.

**Environment (please complete the following information):**
 - **WebdriverIO version:** latest version
 - **Additional wdio packages used (if applicable):** @wdio/mocha-framework
 - 

**Describe the bug**
The jest and mocha taipings are conflicting when using your package. Is it possible to move @ types/mocha to devDependecies?

We use jest for unit testing and and your packages for e2e testing

**To Reproduce**
npm i

**Log**
For example 

```
Property 'each' does not exist on type 'TestFunction'.

39   it.each([true, false])('Should update isRootPreset', async isRootPreset => {
```

> Please use the correct markdown

**Comment by christian-bromann** [comment]
@PiQx mind making a PR?

**Comment by PiQx** [comment]
> @PiQx mind making a PR?

nope, i can

**Comment by christian-bromann** [comment]
fixed in #6967
